File: tasks/translation/output/Qwen/Qwen2.5-Coder-32B-Instruct-AWQ/codenet/cot/Java/C/s780275612_comments_removed.c
#include <stdio.h>
#include <math.h>

int main() {
    int intA;
    scanf("%d", &intA);

    double dbAncer = pow(intA, 3);
    int intAncer = (int)dbAncer;
    printf("%d\n", intAncer);

    return 0;
}
// End of Code

File: tasks/translation/output/Qwen/Qwen2.5-32B-Instruct-AWQ/codenet/cot/Java/Go/s780275612_comments_removed.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var intA int
	fmt.Scanf("%d", &intA)

	dbAncer := math.Pow(float64(intA), 3)
	intAncer := int(dbAncer)
	fmt.Println(intAncer)
	// End of Code
